py/Boinc/setup_project.py

```diff
@@ -339,6 +340,31 @@ def dir(*dirs):
             shutil.copy(srcdir('html/user/sample_motd.php'), dir('html/user/motd.php'))
         os.system("rm -f "+dir('html/languages/translations/*'))
         install_glob(srcdir('html/languages/translations/*.po'), dir('html/languages/translations/'))
+        try:
+            os.remove(srcdir('html/inc/release.inc'))
+        except OSError:
+            pass
+        try:
+            s = subprocess.Popen(["git", "rev-parse", "HEAD"], stdout=subprocess.PIPE, stderr=open(os.devnull, 'w'))
+            commit = s.stdout.read()[:7]
+            s = subprocess.Popen(["git", "branch"], stdout=subprocess.PIPE, stderr=open(os.devnull, 'w'))
+            branch = s.stdout.read().split('*')[1].split('\n')[0].strip()
+            s = subprocess.Popen(["git", "tag", "-l", "--points-at", "HEAD"], stdout=subprocess.PIPE, stderr=open(os.devnull, 'w'))
```

Comment:
Might be best to ensure you're looking only at "server_release" tags here, which I imagine is what you want. You could do `git describe --match "server_release/*" --tags --exact HEAD`. This will error if the current commit doesn't have such a tag, in which case you could leave it blank, or you might use the output of `git describe --match "server_release/*" --tags --abbrev=0  HEAD` and append a `-DEV` or something like that so you get a version which looks like `server_release/0.9-DEV`, which would indicate the most recent server_release tag is 0.9 but you're some number of commits ahead.